VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.html

```diff
@@ -128,66 +128,42 @@ <h2 class="govuk-heading-s">Participants ({{ nonJudgeParticipants.length }})</h2
 
-          <div class="govuk-body-s govuk-!-margin-bottom-3">
-            <label
-              id="p-{{ participant.id }}-status-participant"
-              class="govuk-body-s label-status label-status--{{ getParticipantStatusCss(participant) }}"
-            >
-              {{ getParticipantStatus(participant) }}
-            </label>
-          </div>
+          <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
+            <span *ngIf="(participant)?hasInterpreterLink(participant):false" class="vh-interpreter-icon">
+              <em class="far fa-2x fa-language"></em>
+            </span>
+            {{ participant.name }}
+          </p>
 
-          <div class="vh-line" *ngIf="!lastItem"></div>
-        </div>
-      </div>
-    </div>
+          <p id="p-{{ participant.id }}-hearing-role-participant" class="govuk-body-m govuk-!-margin-bottom-3">
+            {{ participant.hearing_role }}
+          </p>
 
-    <div *ngIf="!litigantInPerson" class="vh-border">
-      <div *ngFor="let participant of nonJudgeParticipants; let lastItem = last" class="govuk-grid-row govuk-!-margin-bottom-1">
-        <div id="p-row-{{ participant.id }}-participant" class="govuk-grid-column-full govuk-!-margin-top-2">
-          <!-- represented by solicitor -->
-          <div *ngIf="!litigantInPerson">
-            <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
-              {{ participant.name }}
-            </p>
-            <p id="p-{{ participant.id }}-representee" *ngIf="participant.representee" class="govuk-body-m govuk-!-margin-bottom-3">
-              {{ !isCaseTypeNone(participant) ? 'Representative' : participant.hearing_role }} for {{ participant.representee }}
-            </p>
-            <p
-              id="p-{{ participant.id }}-hearing-role-participant"
-              *ngIf="!participant.representee"
-              class="govuk-body-m govuk-!-margin-bottom-3"
-            >
-              {{ participant.hearing_role }}
-            </p>
-            <p
-              *ngIf="!isCaseTypeNone(participant)"
-              id="p-{{ participant.id }}-case-type-group-participant"
-              class="govuk-body-m govuk-!-margin-bottom-3"
-            >
-              {{ participant.case_type_group }}
-            </p>
-          </div>
+          <p id="p-{{ participant.id }}-hearing-role-participant" *ngIf="(!participant.representee)?isInterpreter(participant):false"
+            class="govuk-body-s govuk-!-margin-bottom-1">
+            {{ participant.hearing_role }} for <br><strong>{{ getInterpreteeName(participant.id) }}</strong>
+          </p>
+
+          <p id="p-{{ participant.id }}-representee-participant" *ngIf="participant.representee" class="govuk-body-s govuk-!-margin-bottom-1">
+            {{ !isCaseTypeNone(participant) ? 'Representative' : participant.hearing_role }} for <br><strong>{{ participant.representee }}</strong>
```

Comment:
just reverse the true/false expressions and remove the !

Reply:
not code I have touched, its only highlighted because I amended the HTML to reduce nesting

Reply:
fixed it anyway, as correct logic makes more sense